core/services/ocr2/delegate.go

```diff
@@ -98,8 +98,7 @@ func (d Delegate) ServicesForSpec(jobSpec job.Job) ([]job.ServiceCtx, error) {
 		return nil, errors.New("peerWrapper is not started. OCR2 jobs require a started and running peer. Did you forget to specify P2P_LISTEN_PORT?")
 	}
 
-	lggr := d.lggr.With(
-		"OCRLogger", "true",
+	lggr := d.lggr.Named("OCR").With(
```

Comment:
I think these just predated the `Named` logger convention - I didn't see any other uses of this `OCRLogger: true` pattern.

Reply:
@kaleofduty wdyt about this? It is more conventional, but don't want to surprise you by breaking existing log filters.

Reply:
Thanks for checking. The proposed change shouldn't pose a problem from the research team's perspective.